component/conprof/scrape/ticker.go

```diff
@@ -99,6 +100,7 @@ func (t *Ticker) run(ctx context.Context) {
 func (t *Ticker) notify(now time.Time) {
 	t.Lock()
 	defer t.Unlock()
+	t.lastTime = now
```

Comment:
why a now need to be passed in? How about simply `t.lastTime = time.Now()`?

Reply:
The time `now` is got from the ticker, avoid to call `time.Now()` again here.